server/public/model/user.go

```diff
@@ -472,6 +479,14 @@ func (u *User) PreSave() {
 	if u.Password != "" {
 		u.Password = HashPassword(u.Password)
 	}
+
+	if u.Props != nil {
```

Comment:
Is this condition needed with `u.Props = make(map[string]string)` above?

Reply:
You are correct...